errors/errors.go

```diff
@@ -10,114 +10,114 @@ import (
 
 // start of Placementgroup Messages
 
-const GetPlacementGroupOperationFailed = "failed to perform Get Placement Group Operation for placement group %s with error %w"
-const CreatePlacementGroupOperationFailed = "failed to perform Create Placement Group Operation for cloud instance %s with error  %w"
-const DeletePlacementGroupOperationFailed = "failed to perform Delete Placement Group Operation for placement group %s with error %w"
-const AddMemberPlacementGroupOperationFailed = "failed to perform Add Member Operation for instance %s and placement group %s with error %w"
-const DeleteMemberPlacementGroupOperationFailed = "failed to perform Delete Member Operation for instance %s and placement group %s with error %w"
+const GetPlacementGroupOperationFailed = "failed to perform Get Placement Group Operation for placement group %s with error %v"
```

Comment:
We have added wrapping errors because we unwrap it in the Terraform code to check for 404 and other response codes.

Reply:
I changed it because it was giving me a warning. I was unsure how else to resolve it.
<img width="665" alt="Screenshot 2023-09-22 at 8 23 59 AM" src="https://github.com/IBM-Cloud/power-go-client/assets/90468712/c2033055-1600-4bfc-8c17-db4f9d56759a">

Reply:
Then I'd suggest to not send the string as arg to the util funct, instead just pass the err to determine the 5XX code.

Reply:
ok I'll revert it. 👍